### PR TITLE
[FSDE-28508] Increase margin in radio button component

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

File: src/components/Radio/Radio.js [hunk 1]
```diff
@@ -38,12 +41,11 @@ export default class Radio extends React.Component {
       .forEach((_, i) => {
         if (this.props[`option${i}`]) {
           const uuid = `${this.state.id}-${i}`;
-          const inline = this.props.inline ? '-inline' : '';
 
           radiosToRender.push(
-            <div className={`chi-form__item ${inline}`} key={`radio-${i}`}>
-              <div className="chi-radio">
-                <input className="chi-radio__input" type="radio" name={`radios-${uuid}`} id={uuid}
+            <div className={this.props.inline ? `${FORM_CLASSES.ITEM} ${INLINE_CLASS}` : 'chi-col -w--12 -mb--1'} key={`radio-${i}`}>
```
Comment (josecarlosgonzalezv) [inline src/components/Radio/Radio.js:46]: We have already defined some of these utility classes.

```suggestion
            <div className={this.props.inline ? `${FORM_CLASSES.ITEM} ${INLINE_CLASS}` : `${UTILITY_CLASSES.COLUMN} ${UTILITY_CLASSES.MARGIN.BOTTOM[1]} -w--12`} key={`radio-${i}`}>
```

File: src/components/Radio/Radio.js [hunk 2]
```diff
@@ -80,14 +82,20 @@ export default class Radio extends React.Component {
       </div> : '';
 
     const fieldLabel = this.props.fieldLabel ?
-      <div className={`${LABEL_CLASSES.LABEL}`}>
-        {this.props.fieldLabel}
-        {message}
-        {info}
-      </div> : '';
+        <div className={`${LABEL_CLASSES.LABEL}`}>
```
Comment (josecarlosgonzalezv) [inline src/components/Radio/Radio.js:85]: You can do this with similar cases.

```suggestion
        <div className={LABEL_CLASSES.LABEL}>
```

File: src/components/Radio/Radio.js [hunk 3]
```diff
@@ -80,14 +82,20 @@ export default class Radio extends React.Component {
...
+          {info}
+        </div>
+      : '';
+
     return (
       <fieldset>
-        {fieldLabel}
+        {this.props.inline
+          ? fieldLabel
+          : <div className="chi-col -w--12 -mb--1">
```
Comment (josecarlosgonzalezv) [inline src/components/Radio/Radio.js:96]: Use the defined utility classes.

File: src/components/Radio/Radio.js [hunk 4]
```diff
@@ -80,14 +82,20 @@ export default class Radio extends React.Component {
...
+              {fieldLabel}
+            </div>}
```
Comment (josecarlosgonzalezv) [inline src/components/Radio/Radio.js:98]: `</div>}` is strange, could you format the whole file to see if you can fix it?

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]
On line 40 you have `Array(11).fill()`, please make 11 a constant.

**Review by jllr — APPROVED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Pull request merged**